content/internal/home.pyx

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2020  seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+import string
+import os
+import time
+import subprocess
+from mod_python import apache, util
+from ertos import *
+from ertos_config import cfg
+
+# We assume people who can get to this point are more-or-less trusted; there's
```

Comment:
Yes, assuming auth is already taken care of, it looks OK.

Things like `subprocess.run("cd %s; ..." % path` would be a worry otherwise :wink: